_overviews/scala3-macros/tutorial/reflection.md

```diff
@@ -75,6 +98,47 @@ def g[T: Type](using Quotes) =
   ...
 ```
 
+## Symbols
+
+The APIs of `Term` and `TypeTree` are relatively *closed* in the sense that methods produce and accept values
+whose types are defined in the API. You might notice however the presence of `Symbol`s which identify definitions.
+
+Each `Term` or `TypeRepr` (therefore `Expr` and `Type`) have an associated symbol.
+`Symbol` exposes and is used by many useful methods for example:
+
+ - `declaredFields` and `declaredMethods` allow you to iterate on the fields and members defined inside a symbol
+ - `flags` allow you to check multiple properties of a symbol
+ - `companionObject` and `companionModule` provide a way to jump to and from the companion object/class.
+ - `TypeRepr.baseClasses` returns the list of symbols of classes extended by a type. 
+ - `Symbol.pos` gives you access to the position where the symbol is defined, the source code of the definition
+ and even the filename where the symbol is defined.
+ - and many useful others that you can find in `SymbolMethods`
+
+### To Symbol and back
+
+ - `TypeRepr.typeSymbol` returns the symbol of the type
+ - `TypeRepr.termSymbol` returns the symbol of the term associated to the type.
+ - `TypeRepr.memberType(symbol)` returns the `TypeRepr` of the provided symbol
+ - `TypeRepr.of[T].typeSymbol` is the recommended way to obtain a `Symbol` given a `Type[T]`
+ - `Tree.symbol` returns the symbol associated to a tree. Given that `Term <: Tree`,
+ `Expr.asTerm.symbol` is the best way to obtain the symbol associated to an `Expr[T]`
+
+ - `Symbol.tree` returns the `Tree` associated to the symbol. Be extremely careful when using this
+ method as the tree for a symbol might not be defined. When the code associated to the symbol
+ is defined in a different moment than this access, if the `Yretain-trees` compilation option
+ is not used, then the `tree` of the symbol will not be available. Symbols originated from
+ Java code do not have an associated `tree`.
+
+## Suggestion and anti-patterns
+
+ - Avoid using `TypeTree`s (therefore `TypeTree.of`) when you could use `TypeRepr`
+ - Avoid using `Symbol.tree` for the reasons mentioned [here](#symbols) and because of
```

Comment:
Did you check that this anchor link works?

Reply:
Yes it seems to work 
![image](https://user-images.githubusercontent.com/6325220/117622539-fcc3a380-b172-11eb-879c-df6101c399ac.png)